angular-workspace/package.json

```diff
@@ -5,15 +5,18 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
-    "build": "npm run build:library && npm run build:application",
+    "build": "npm run build:library && npm run build:spright && npm run build:application",
     "build:library": "npm run generate-icons && ng build @ni/nimble-angular",
```

Comment:
I'd lean towards renaming the `*:library` scripts to `*:nimble`. Feels asymmetric to use "library" now that there are multiple libs.